test/support/utils.h

```diff
@@ -36,6 +36,9 @@
 
 // Test data ranges other than those that start at the beginning of an input.
 const int max_n = 100000;
+
+// All these offset consts used for indirect testing of specialization for hetero
```

Comment:
"...testing of calculation an offset parameter (as a result `dpl::begin(buf) + offset`) for further passing within sycl::accessor constructor."

Reply:
Fixed